2017-09-14 21:51:42,769Â 13376 DEBUG pytest.py:18 Information during calling f()
